HomeWork/src/main/java/arraylist/ArrayList.java

```diff
@@ -0,0 +1,87 @@
+package arraylist;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+
+public class ArrayList {
+    public static void main(String[] args) {
+        String[] str = {"A", "B", "C", "D"};
+        Integer[] integers = {1, 3, 4, 5, 3, 21};
+        List<Integer> list = new java.util.ArrayList<>(Arrays.asList(1, 3, 2, 7, 3, 4, 4, 10, 4, 6, 6));
+        List<String> stringList = new java.util.ArrayList<>(Arrays.asList("fox", "list", "fox", "apple", "list", "cat",
+                "orange", "orange", "dota", "Alex", "fox", "list", "apple", "blue"));
+    }
+
+    static public List<String> findOccurrence(List<String> elements) {
```

Comment:
Please never declare modifiers like this: `static public`. Would be better and more common to do it like this: `public static`